app/views/notes/_card.html.erb

```diff
@@ -13,11 +13,11 @@
     <% if node.status == 4 %>
       <p class="alert alert-warning moderated">
         <% if @current_user && ['admin','moderator'].include?(@current_user.role) %>
-          <%= t('notes._notes.moderate_first_time_post') %> <br class="d-none d-lg-inline"/>
-          <a class="btn btn-outline-secondary btn-sm" href="/moderate/publish/<%= node.id %>"><%= t('notes._notes.approve') %></a>
-          <a class="btn btn-outline-secondary btn-sm" data-test="spam" href="/moderate/spam/<%= node.id %>"><%= t('notes._notes.spam') %></a>
+          <%= translation('notes._notes.moderate_first_time_post') %> <br class="d-none d-lg-inline"/>
```

Comment:
OK, could we try this? 

```suggestion
          <%= ApplicationController.helpers.translation('notes._notes.moderate_first_time_post') %> <br class="d-none d-lg-inline"/>
```
And repeat that change for all instances of `translation()` in this file? 

What i think is happening is that this template file is (uniquely, we don't do this for other templates) being rendered inside the `/app/models/concerns/node_shared.rb` file as we've modularized the generation of this code a bit. And in model or concern code, we don't have "quick access" to view helpers. We need instead to use a full reference with `ApplicationController.helpers._______` to get to them. 

Let's make this change throughout this one file and see if it passes tests then!

Reply:
Okay, got it. Shall I make the changes?

Reply:
Yes @Priyaraj17, please make the changes...

Reply:
Hey, @RuthNjeri  @jywarren 
I have made the changes. I hope this will be fine.

Reply:
Oof, that didn't work, did it? Gosh, sorry @priyaraj17 - i think we need to look at the error logs and potentially look at other options for how to use the helpers:

https://stackoverflow.com/questions/6820547/use-a-rails-helper-in-a-model

Reply:
Yeah, It didn't work. In fact, I created a PR in the public lab editor repo a couple of weeks ago. A few tests were failing in that PR too.

Reply:
Should we also check if `current_user` exists?